src/libraries/System.Private.CoreLib/src/System/SearchValues/IndexOfAnyAsciiSearcher.cs

```diff
@@ -17,18 +17,36 @@ internal static class IndexOfAnyAsciiSearcher
     {
         public struct AsciiState(Vector128<byte> bitmap, BitVector256 lookup)
         {
-            public Vector256<byte> Bitmap = Vector256.Create(bitmap);
+            public Vector512<byte> Bitmap512 = Vector512.Create(bitmap);
             public BitVector256 Lookup = lookup;
 
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public readonly Vector128<byte> Bitmap128() => Bitmap512._lower._lower;
```

Comment:
Nit: why a method rather than a property?

Reply:
It's a bit more code to define the `get` since we need the `MethodImpl` on it. No other reason.